TreeDB/cmd/treedb_rag_benchmark/minima.go

```
@@ -0,0 +1,1476 @@
+}
+
+func minimaStateDocuments(documents []minimaGeneratedDocument) []minimaStateDocument {
+	out := make([]minimaStateDocument, len(documents))
+	for i, document := range documents {
+		out[i] = minimaStateDocument{ID: document.ID, Content: document.Content, UserID: document.UserID, FPath: document.FPath}
+	}
+	return out
+}
+
+func minimaDigest(value any) string {
+	raw, _ := json.Marshal(value)
```

golangci-lint (2.12.2)

[error] Error return value of `encoding/json.Marshal` is not checked: unsafe type `any` found

(errchkjson)